app/models/dog.rb

```diff
@@ -7,4 +7,335 @@ class Dog < ActiveRecord::Base
+    "Maltichon",
+    "Maltipoo",
+    "Maltzu",
+    "Manchester Terrier",
+    "Maremma Sheepdog",
+    "Mastador",
+    "Mastiff",
+    "Mini Bernedoodle",
+    "Miniature Australian Shepherd",
+    "Miniature Bull Terrier",
+    "Miniature Dachschund",
+    "Miniature Pinscher",
+    "Miniature Poodle",
+    "Miniature Schnauzer",
+    "Morkie",
+    "Muggin",
+    "Neapolitan Mastiff",
+    "New Zealand Huntaway",
+    "Newfoundland",
+    "Norfolk Terrier",
+    "Northern Inuit",
+    "Norwegian Buhund",
+    "Norwegian Elkhound",
+    "Norwegian Lundehund",
+    "Norwich Terrier",
+    "Nova Scotia Duck Tolling Retriever",
+    "Old English Sheepdog",
+    "Olde English Bulldogge",
+    "Ori Pei",
+    "Otterhound",
+    "Papichon",
+    "Papillon",
+    "Parson Russell Terrier",
+    "Patterdale Terrier",
+    "Peek-a-Pom",
+    "Peekapoo",
+    "Pekingese",
+    "Petit Basset Griffon Vendeen",
+    "Pharaoh Hound",
+    "Plott",
+    "Pointer",
+    "Polish Lowland Sheepdog",
+    "Pom Chi",
+    "Pomapoo",
+    "Pomeranian",
+    "Pomino",
+    "Poodle",
+    "Poogle",
+    "Portuguese Podengo Pequeno",
+    "Portugese Water Dog",
+    "Prague Ratter",
+    "Presa Canario",
+    "Pug",
+    "Pugapoo",
+    "Puggle",
+    "Pugshire",
+    "Puli",
+    "Pumi",
+    "Pyrenean Shepherd",
+    "Rat Terrier",
+    "Ratonero Bodeguero Andaluz",
+    "Redbone Coonhound",
+    "Rhodesian Ridgeback",
+    "Rottweiler",
+    "Russell Terrier",
+    "Saint Berdoodle",
+    "Saluki",
+    "Samoyed",
+    "Samusky",
+    "Schipperke",
+    "Schipperpoo",
+    "Schnoodle",
+    "Schweenie",
+    "Scotchie",
+    "Scottish Deerhound",
+    "Scottish Terrier",
+    "Sealyham Terrier",
+    "Shar-Poo",
+    "Sheltiedoodle",
+    "Shetland Sheepdog",
+    "Shichon",
+    "Shih-poo",
+    "Shikoku Ken",
+    "Shiloh Shepherd",
+    "Shinese",
+    "Shiranian",
+    "Shorkie Tzu",
+    "Siberian Retriever",
+    "Silken Windhound",
+    "Silkshire Terrier",
+    "Silky Terrier",
+    "Skye Terrier",
+    "Sloughi",
+    "Smooth Collie",
+    "Smooth Fox Terrier",
+    "Snorkie",
+    "Soft Coated Wheaten Terrier",
+    "Spanish Water Dog",
+    "Spinone Italiano",
+    "St. Bernard",
+    "Staffordshire Bull Boxer",
+    "Staffordshire Bull Terrier",
+    "Standard Poodle",
+    "Standard Schnauzer",
+    "Sussex Spaniel",
+    "Swedish Vallhund",
+    "Tamaskan",
+    "Tibalier",
+    "Tibetan Mastiff",
+    "Tibetan Spaniel",
+    "Tibetan Terrier",
+    "Toy Fox Terrier",
+    "Toy Poodle",
+    "Treeing Walker Coonhound",
+    "Valley Bulldog",
+    "Vizsla",
+    "Weimaraner",
+    "Welsh Springer Spaniel",
+    "Welsh Terrier",
+    "West Highland White Terrier",
+    "Westiepoo",
+    "Westion",
+    "Whippet",
+    "White Shepherd",
+    "Whoodle",
+    "Wire Fox Terrier",
+    "Wirehaired Pointing Griffon",
+    "Wirehaired Vizsla",
+    "Xoloitzcuintli",
+    "Yo-Chon",
+    "Yorkie Pom",
+    "Yorkiepoo",
+    "Yorkshire Terrier"
+    ]
```

Comment:
Indent the right bracket the same as the start of the line where the left bracket is.